app/controllers/api/v1/domain_blocks_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Api::V1::DomainBlocksController < ApiController
+  before_action -> { doorkeeper_authorize! :follow }
+  before_action :require_user!
+
+  respond_to :json
+
+  def show
```

Comment:
Because it's a "resource" route and not "resources" (so that we can send DELETE /api/v1/domain_blocks)